pkg/meta/base.go

```diff
@@ -127,6 +138,77 @@ func (r *baseMeta) newMsg(mid uint32, args ...interface{}) error {
 	return fmt.Errorf("message %d is not supported", mid)
 }
 
+func (m *baseMeta) Load() (*Format, error) {
+	body, err := m.en.doLoad()
+	if err == nil && len(body) == 0 {
+		err = fmt.Errorf("database is not formatted")
+	}
+	if err != nil {
+		return nil, err
+	}
+	if err = json.Unmarshal(body, &m.fmt); err != nil {
+		return nil, fmt.Errorf("json: %s", err)
+	}
+	return &m.fmt, nil
+}
+
+func (m *baseMeta) NewSession() error {
+	go m.refreshUsage()
+	if m.conf.ReadOnly {
+		return nil
+	}
+
+	v, err := m.en.incrCounter("nextSession", 1)
+	if err != nil {
+		return fmt.Errorf("get session ID: %s", err)
+	}
+	m.sid = uint64(v)
+	info := newSessionInfo()
+	info.MountPoint = m.conf.MountPoint
+	data, err := json.Marshal(info)
+	if err != nil {
+		return fmt.Errorf("json: %s", err)
+	}
+	if err = m.en.doNewSession(data); err != nil {
+		return fmt.Errorf("create session: %s", err)
+	}
+	logger.Debugf("create session %d OK", m.sid)
+
+	go m.refreshSession()
+	go m.cleanupDeletedFiles()
+	go m.cleanupSlices()
+	go m.cleanupTrash()
+	return nil
+}
+
+func (m *baseMeta) refreshSession() {
+	for {
+		time.Sleep(time.Minute)
+		m.Lock()
+		if m.umounting {
+			m.Unlock()
+			return
+		}
+		m.en.doRefreshSession()
+		m.Unlock()
+		if _, err := m.Load(); err != nil {
+			logger.Warnf("reload setting: %s", err)
+		}
+		go m.CleanStaleSessions()
```

Comment:
if m.en.setIfSmall(...) {
  go m.CleanStaleSessions()
}

Reply:
Will do in the next PR